otter/integration/tests/test_convergence.py

```diff
@@ -67,6 +68,54 @@ def _check_fds(_):
             return deferLater(reactor, 0, _check_fds, None)
         return self.pool.closeCachedConnections().addBoth(_check_fds)
 
+    def test_reaction_to_oob_deletion_after_scale_up(self):
+        """Exercise out-of-band server deletion, but then scale up afterwards.
+        The goal is to spin up, say, three servers, then use Nova to delete
+        one of them directly, without Autoscale's knowledge.  Then we scale up
+        by, say, two servers.  If convergence is working as expected, we expect
+        four servers at the end.
```

Comment:
I think this is off:

3 servers running
-1 deleted OOB
+2 from policy

Convergence should fix the deletion by replacing it, _and_ add 2 for the policy execution, to arrive at 5, not 4.